progress/progress.go

```diff
@@ -0,0 +1,91 @@
+package progress
+
+import (
+	"strings"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/lucasb-eyer/go-colorful"
+	"github.com/muesli/termenv"
+)
+
+// Value is explicit type of, you know, progression. Can be 0.0 < x < 1.0
+type Value float64
+
+type Model struct {
+	// Left side color of progress bar. By default, it's #00dbde
+	StartColor colorful.Color
+
+	// Left side color of progress bar. By default, it's #fc00ff
+	EndColor colorful.Color
+
+	// Length of progress bar in symbols.
+	Length int
```

Comment:
I'd change this to `Width` for consistency with other packages in this repo.

Reply:
Yeah, i agree